src/test/pybind/test_rados.py

```diff
@@ -34,8 +34,11 @@ def test_rados_init():
 
 def test_ioctx_context_manager():
     with Rados(conffile='', rados_id='admin') as conn:
+        if not conn.pool_exists('rbd'):
```

Comment:
could you help me understand why we could run into this issue given that "rbd" pool is created by https://github.com/ceph/ceph/blob/96b3a8a5ee359a235188866cf8fe11af33253ff7/qa/workunits/rados/test_python.sh#L3 ?

Reply:
Hi @tchaikov , I find that "ceph osd pool create rbd"  was added in Nov 13  2019. I run into this issue in Nov 7 2019. 
The issue was resolved, I will close this PR.  thanks.

Reply:
@zhengchengyao thank you!